README.md

```diff
@@ -88,14 +88,27 @@ SegmentContextPtr current_segment = createSegmentContext(seg_config);
 First, you must create root span to trace current workload.
 
 ```cpp
-CurrentSegmentSpanPtr current_span = current_segment->createCurrentSegmentRootSpan();
+CurrentSegmentSpanPtr current_span = current_segment->createEntrySpan();
```

Comment:
I think `create root span` != `create entry span`, even in many cases they are.

Reply:
So I changed implementation like [this](https://github.com/SkyAPM/cpp2sky/blob/fix-issue-25/source/segment_context_impl.cc#L175-L192). In the [previous implementation](https://github.com/SkyAPM/cpp2sky/blob/main/source/segment_context_impl.cc#L179-L185) `createCurrentSegmentRootSpan` means to create `Entry Span`.

But, `Root Span` has pretty unclear because it has undetectable whether that is `Entry` or `Exit`. Also, `Local` span is not supported in the current implementation but in the future it can be introduced, then `createCurrentSegmentSpan` can be difficult to interpret. So I changed `createCurrentSegmentRootSpan` -> `createEntrySpan` and `createCurrentSegmentSpan` -> `createExitSpan`.

Reply:
Make sense. We don't need root span concept, it should be automatically created based on the `current segment`.
Another naming topic, `current segment` is actually exposed as `tracing context` from my understanding. Segment is the concept when you need to send the set of data to the OAP server.

Reply:
Sounds good. ok I will introduce another PR that renames `current segment` to `tracing context` and automatic creation of entry span.